src/model/remove-life-cycle.ts

```diff
@@ -0,0 +1,22 @@
+import { hooks } from '../utils/hooks';
+import { validateSchema } from '..';
```

Comment:
I didn't realize you could do this, I assume this imports from `../index.ts`. Appears `validateSchema` and `store` are imported but not used (yet?)

Reply:
Fixed: all unused variables were removed

'..' is the same '../index.ts'